content/en/contribute/code/core/dev-environment.md

```diff
@@ -6,6 +6,7 @@ description: >
   Getting your local machine ready to do development work on CHT Core.
 aliases: >
   apps/guides/hosting/core-developer
+  apps/guides/hosting/hosting/4.x/app-developer
```

Comment:
These both need a leading `/` to work. Note that  I had a cached redirect in hugo that I had to clear with `hugo serve --cleanDestinationDir` and then testing these with `curl -v localhost:1313/apps/guides/hosting/hosting/4.x/app-developer/  -o /dev/null` would return a `404`.

```suggestion
  /apps/guides/hosting/core-developer
  /apps/guides/hosting/hosting/4.x/app-developer
```